src/main/java/cz/muni/fi/pv286/mmt/exceptions/InvalidNestingException.java

```diff
@@ -0,0 +1,16 @@
+package cz.muni.fi.pv286.mmt.exceptions;
+
+import java.io.IOException;
+
+/**
+ * An exception that is thrown when a bracket mismatch occurs.
+ */
+public class InvalidNestingException extends IOException {
+    public InvalidNestingException(String message) {
+        super(message);
+    }
+
+    public InvalidNestingException() {
+        super("Invalid nesting");
+    }
+}
```

Comment:
**suggestion**: Remove unused class.

It's a code smell and the code it's harder to read.